commands/next-flight.js

```diff
@@ -1,45 +1,45 @@
-const {SlashCommandBuilder, EmbedBuilder} = require('discord.js');
-const {ActionRowBuilder, ButtonBuilder, ButtonStyle} = require('discord.js');
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js')
+const { ActionRowBuilder, ButtonBuilder, ButtonStyle } = require('discord.js')
 
 module.exports = {
-    data: new SlashCommandBuilder()
-        .setName('next-flight')
-        .setDescription('The link to find out our next flight!'),
-    async execute(interaction) {
+  data: new SlashCommandBuilder()
+    .setName('next-flight')
+    .setDescription('The link to find out our next flight!'),
+  async execute (interaction) {
+    interaction.deferReply()
+    const events = await interaction.guild.scheduledEvents.fetch()
+    const sortedEvents = events.sort((a, b) => a.scheduledStartAt - b.scheduledStartAt)
+    const event = sortedEvents.first()
 
-        const events = await interaction.guild.scheduledEvents.fetch();
-        const sortedEvents = events.sort((a, b) => a.scheduledStartAt - b.scheduledStartAt);
-        const event = sortedEvents.first();
+    if (event === undefined) {
+      await interaction.editReply('No events :(')
+    } else {
+      const eventTime = Date.parse(event.scheduledStartAt) / 1000
```

Comment:
Couple of things here. According to the docs, here: https://old.discordjs.dev/#/docs/discord.js/main/class/GuildScheduledEvent?scrollTo=scheduledStartAt scheduledStartAt could be `null`, but is also already a `Date` object. `Date.parse` on an already existing date object will return `NaN` and the bot will blow up. Also, why are we dividing by 1000? It's not needed for the date format, I don't think. Discord is just an epoch timestamp converter.